### PR TITLE
chore(mprocs): improve `bitcoin-cli` alias

### PR DESCRIPTION
**Pull request by dpc**
I want just `bitcoin-cli getblockcount` to work.

### CODE REVIEW

**Review by bradleystachurski — APPROVED** [review]
👍 
```
bash-5.2$ bitcoin-cli getblockcount
194
```

File: scripts/dev/aliases.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 alias lightning-cli="\$FM_LIGHTNING_CLI"
 alias lncli="\$FM_LNCLI"
-alias bitcoin-cli="\$FM_BTC_CLIENT"
+alias bitcoin-cli="\$FM_BTC_CLIENT -datadir=\$FM_BTC_DIR"
```
Comment (elsirion) [inline scripts/dev/aliases.sh:5]: Shouldn't this be done here? 

https://github.com/fedimint/fedimint/blob/c05926868fc5e983637a25ca879b8e603e055b9d/devimint/src/vars.rs#L137

**Review by elsirion — APPROVED** [review]
If you have a reason to add the argument here feel free to merge, otherwise we might want to add it in `devimint`.

File: scripts/dev/aliases.sh (hunk 1 above)
Comment (Maan2003) [reply scripts/dev/aliases.sh]: done

**Review by elsirion — APPROVED** [review]

**Pull request merged**